src/test/java/kitchenpos/acceptance/MenuAcceptanceTest.java

```diff
@@ -0,0 +1,107 @@
+package kitchenpos.acceptance;
+
+import static kitchenpos.KitchenPosFixtures.까르보치킨_생성요청;
+import static kitchenpos.KitchenPosFixtures.두_마리_메뉴_생성요청;
+import static kitchenpos.KitchenPosFixtures.메뉴_URL;
+import static kitchenpos.KitchenPosFixtures.메뉴그룹_URL;
+import static kitchenpos.KitchenPosFixtures.짜장치킨_생성요청;
+import static kitchenpos.KitchenPosFixtures.프로덕트_URL;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.Product;
+import kitchenpos.ui.dto.response.MenuGroupResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class MenuAcceptanceTest extends AcceptanceTest {
+    private MenuGroupResponse 두_마리_메뉴;
+    private Product 생성된_까르보치킨;
+    private Product 생성된_짜장치킨;
+
+    @BeforeEach
+    void setupMenuGroup() {
+        두_마리_메뉴 = 생성요청(메뉴그룹_URL, 두_마리_메뉴_생성요청).body().as(MenuGroupResponse.class);
+        생성된_까르보치킨 = 생성요청(프로덕트_URL, 까르보치킨_생성요청).body().as(Product.class);
+        생성된_짜장치킨 = 생성요청(프로덕트_URL, 짜장치킨_생성요청).body().as(Product.class);
+    }
+
+    @Test
+    void 신규_메뉴를_생성할_수_있다() {
+        // given & when
+        final var 메뉴명 = "까르보 한 마리 + 짜장 한 마리";
+        final var 메뉴가격 = new BigDecimal(30000);
+        final var 메뉴_생성요청_데이터 = Map.of(
+                "name", 메뉴명,
+                "price", 메뉴가격,
+                "menuGroupId", 두_마리_메뉴.getId(),
+                "menuProducts", List.of(
+                        Map.of("productId", 생성된_까르보치킨.getId(), "quantity", 1),
+                        Map.of("productId", 생성된_짜장치킨.getId(), "quantity", 1)
+                )
+        );
+
+        final var 메뉴_생성응답 = 생성요청(메뉴_URL, 메뉴_생성요청_데이터);
+        final var 생성된_메뉴 = 메뉴_생성응답.body().as(Menu.class);
+        final var 메뉴_내_상품들 = 생성된_메뉴.getMenuProducts();
+
+        // then
+        assertAll(
+                응답일치(메뉴_생성응답, HttpStatus.CREATED),
+                단일_데이터_검증(생성된_메뉴.getId(), 1L),
+                단일_데이터_검증(생성된_메뉴.getName(), 메뉴명),
+                단일_데이터_검증(생성된_메뉴.getPrice().doubleValue(), 메뉴가격.doubleValue()),
+                단일_데이터_검증(생성된_메뉴.getMenuGroupId(), 두_마리_메뉴.getId()),
+                리스트_데이터_검증(메뉴_내_상품들, "menuId", 1L, 1L),
+                리스트_데이터_검증(메뉴_내_상품들, "productId", 생성된_까르보치킨.getId(), 생성된_짜장치킨.getId()),
+                리스트_데이터_검증(메뉴_내_상품들, "quantity", 1L, 1L)
+        );
+    }
+
+    @Test
+    void 전체_메뉴를_조회할_수_있다() {
+        // given
+        final var 메뉴명 = "까르보 한 마리 + 짜장 한 마리";
+        final var 메뉴가격 = new BigDecimal(30000);
```

Comment:
```suggestion
        final var 메뉴가격 = new BigDecimal("30000.00");
```
다음 방식을 사용하면 소수점 지정이 가능해서 
BigDemical.doubleValue()를 쓸 필요가 없더라고요!

Reply:
아니잇?!!!!!!!! ㅠㅠㅠㅠㅠ

Reply:
도입 완료했습니다 감사합니다!